<?xml version="1.0"?>
<launch>
    <arg name="threshold_deg" default="30" />
    <arg name="counter_threshold" default="10" />
    <node name="gallery_tracking_node" pkg="gallery_tracking" type="gallery_tracking_node_v2.py"
        output="screen">
        <param name="~threshold_deg" value="$(arg threshold_deg)" type="double" />
        <param name="~counter_threshold" value="$(arg counter_threshold)" type="int" />
    </node>
</launch>